iaso/tests/api/test_tasks.py

```diff
@@ -0,0 +1,56 @@
+from beanstalk_worker.services import TestTaskService
+from iaso.test import APITestCase
+from ...models import OrgUnit, OrgUnitType, Account, Project, SourceVersion, DataSource, Task
+
+
+class IasoTasksTestCase(APITestCase):
+    @classmethod
+    def setUp(cls):
+        source = DataSource.objects.create(name="Valley Championship")
+        old_version = SourceVersion.objects.create(number=1, data_source=source)
+        cls.new_version = SourceVersion.objects.create(number=2, data_source=source)
+
+        account = Account(name="Cobra Kai", default_version=cls.new_version)
+        account.save()
+
+        cls.project = Project(name="The Show", app_id="com.cobrakai.show", account=account)
+        cls.project.save()
+
+        unit_type = OrgUnitType(name="Dojo", short_name="dojo")
```

Comment:
for me unit_type is ambiguous, could you replace by **org_unit_type**?

Reply:
I made the modification